scripts/histmakers/w_z_gen_dists.py

```diff
@@ -171,7 +174,7 @@ def build_graph(df, dataset):
         axis_ewMll = hist.axis.Variable(massBins, name = "ewMll", underflow=False)
         axis_ewPtll = hist.axis.Variable(common.ptV_binning, underflow=False, name = "ewPTll") 
         axis_ewAbsYll = hist.axis.Regular(50, 0, 5, name = "ewAbsYll")
-        df = theory_tools.define_dressed_vars(df, mode="wmass" if isW else "dilepton")
+        df = theory_tools.define_dressed_vars(df, mode="wmass" if isW else "wlike")
```

Comment:
Most of the time when people run the gen histmaker I would expect them to want dilepton observables.